modules/bloop-rifle/src/main/scala/scala/build/blooprifle/internal/Operations.scala

```diff
@@ -349,6 +349,12 @@ object Operations {
           stop0,
           interactiveSession = false
         )
+        if (retCode != 0)
+          logger.error(
+            s"""Bloop bsp command exited with code $retCode. Something may be wrong with the current configuration.
```

Comment:
NIT Last small change
```suggestion
            s"""Bloop 'bsp' command exited with code $retCode. Something may be wrong with the current configuration.
```

(This conveys better that the command is `bsp`.)

Reply:
Thanks for review!